### PR TITLE
feat: Add Non-OS Package support for GCR, DockerV2, and DockerHub

### PR DESCRIPTION
**Pull request by andrewelizondo**
Dependant on this PR being merged upstream.

https://github.com/lacework/go-sdk/pull/539

Signed-off-by: Andre Elizondo <andre@lacework.com>

### CODE REVIEW

File: lacework/resource_lacework_integration_gcr.go [hunk 1]
```diff
@@ -156,6 +156,11 @@ func resourceLaceworkIntegrationGcr() *schema.Resource {
 				Optional: true,
 				Default:  5,
 			},
+			"non_os_package_eval": {
```
Comment (afiune) [inline lacework/resource_lacework_integration_gcr.go:159]: let's call the new argument **non_os_package_support** similar like the UI shows it:
 
```suggestion
			"non_os_package_support": {
```

Comment (andrewelizondo) [reply lacework/resource_lacework_integration_gcr.go]: Totally agree, updated 👍

**Review by afiune — COMMENTED** [review]
Docker Hub is missing adding the field `non_os_package_support` to the Schema.

tenor-151261495

**Comment by afiune** [comment]
Make it soooo! 💯

**Comment by afiune** [comment]
make it so

**Comment by afiune** [comment]
Make it so! 💯

**Comment by afiune** [comment]
make it so

**Comment by afiune** [comment]
make it so! 💯

**Review by afiune — APPROVED** [review]
tenor-151032375

**Pull request merged**